app/Http/Controllers/Api/ActionController.php

```diff
@@ -43,9 +43,9 @@ public function getActions()
         $actions->search(Binput::except(['sort', 'order', 'per_page']));
 
         if ($createdAfter = Binput::get('created_after')) {
-            $actions->whereDate('created_at', '>', Carbon::createFromFormat('Y-m-d', $createdAfter)->toDateString());
+            $actions->whereDate('created_at', '>=', Carbon::createFromFormat('Y-m-d', $createdAfter)->toDateString());
         } elseif ($createdBefore = Binput::get('created_before')) {
-            $actions->whereDate('created_at', '<', Carbon::createFromFormat('Y-m-d', $createdAfter)->toDateString());
+            $actions->whereDate('created_at', '<=', Carbon::createFromFormat('Y-m-d', $createdAfter)->toDateString());
```

Comment:
This is just confusing. If you put exactly the same date for 'after' or 'before' and something was made on _that date_, you end up with that same item in both ranges. Not really... Expected?

idk, seems weird. Maybe just a wording issue, ie 'on or after' + 'before'.